### ISSUE TITLE
Incorporate ability to make discussion more productive

### ISSUE THREAD

**Issue by dijonkitchen** [q2a/question2answer#774]
Might be good to have good defaults like mentioned in this talk: https://devonzuegel.com/post/the-hard-parts-of-open-source-by-evan-czaplicki

**Comment by pupi1985** [comment]
Can you summarize the talk and turn it into concrete setting and default Q2A options? Ideally, into a pull request :)

**Comment by dijonkitchen** [comment]
Best perhaps from this point on in video: https://youtu.be/o_4EX4dPppA?t=2433

Slide 60 onward: https://prezi.com/oowcpzsnwp-8/the-hard-parts-of-open-source/

**Comment by phy25** [comment]
@dijonkitchen Unfortunately I was not able to open the slide on Chrome or Firefox.

As @pupi1985 mentioned, can you summarize the talk and turn it into concrete setting and default Q2A options? Ideally, into a pull request :)

**Comment by svivian** [comment]
Not sure there's really anything actionable here. If you want a user to add more information like the background to a question you can add notices/rules in a widget, or perhaps set up a plugin to capture that extra information.
